tests/data/.sbws.ini

```diff
@@ -0,0 +1,30 @@
+[general]
+# Days into the past that measurements are considered valid
+data_period = 1
+
+[paths]
+sbws_home = .sbws
```

Comment:
Perhaps we don't need to hide this data in .sbws? I'm not used to files in source code being hidden. Why are we doing it here?